frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1130,10 +1139,8 @@ pub mod pallet {
 		Rewarded { account: <T as frame_system::Config>::AccountId, value: BalanceOf<T> },
 		/// An account has been slashed for submitting an invalid signed submission.
 		Slashed { account: <T as frame_system::Config>::AccountId, value: BalanceOf<T> },
-		/// The signed phase of the given round has started.
-		SignedPhaseStarted { round: u32 },
```

Comment:
These changes might break some UIs depending on these events. May be we should label it as `breaking changes` (`E5`?, `B7` ) with some release notes in description. 

@kianenigma might have better insights.

Reply:
I don't think any UI reads these events, so it should be okay.

Nonetheless, I would love to see Ankan's suggestions in PRs that actually do break UIs that we care about. 

The broader meta-discussion here is what degree of release-note do we want to have for breaking frame changes. 

I think there is a LOT LOT more that we can do here, but the current labels are barely enough to reflect changes that are relevant to parachain teams and polkadot, and for this audience I don't think this matters.

Reply:
hi,
currently the only thing you can do to highlight a code change on the release notes is elevating its upgrade criticality with the `C*` labels. This will change though, we're slowly implementing new labels on polkadot/substrate/cumulus that have a clear description and in future the `C*` labels should be used in combination with other labels so that you can distinguish what it is about upfront. Here is the documentation of the new labels explaining how they should be used: https://github.com/paritytech/labels/blob/main/docs/doc_cumulus.md